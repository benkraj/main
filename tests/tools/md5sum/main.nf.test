nextflow_pipeline {

    name "Test --tools md5sum"
    script "main.nf"

    test("Should run without failures") {

        when {
            params {
                outdir = "$outputDir"
                test = true
                tools = 'md5sum'
            }
        }

        then {
            assert workflow.success

            assert snapshot(
                "$outputDir/md5sum/test_1.fastq/test_1.fastq.gz.md5",
                "$outputDir/md5sum/test_2.fastq/test_2.fastq.gz.md5",
                "$outputDir/md5sum/versions.yml"
            ).match()
        }
    }
}
