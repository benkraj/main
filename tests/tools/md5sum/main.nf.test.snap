{
    "Should run without failures": {
        "content": [
            "test_1.fastq.gz.md5:md5,63a3f187d06ae5cffe77fcd39bbdd63a",
            "test_2.fastq.gz.md5:md5,055dc46636836dbd1d9ede2e6ce8cd4e",
            "versions.yml:md5,332de5d2686783d747259535800a11ec"
        ],
        "timestamp": "2023-02-09T13:57:05+0000"
    }
}